src/hotspot/cpu/riscv/stubGenerator_riscv.cpp

```diff
@@ -296,6 +299,16 @@ class StubGenerator: public StubCodeGenerator {
     __ fsd(f26, f26_save);
     __ fsd(f27, f27_save);
 
+    __ frrm(t0);
+    __ sd(t0, frm_save);
+    // Set fcsr to the state we need. We do want Round to Nearest. We
+    // don't want non-IEEE rounding modes.
+    Label skip_fsrmi;
+    __ mv(t1, __ RoundingMode::rne);
+    __ beq(t0, t1, skip_fsrmi);
```

Comment:
You can take advantage of `RoundingMode::rne == 0` by doing `__ beq(t0, zr, skip_fsrmi)` and remove the above `__ mv(t1, __ RoundingMode::rne);`. Please add a `guarantee(__ RoundingMode::rne == 0)` as well, just to self-document.

Reply:
Thanks, fixed.